inc/reception.class.php

```diff
@@ -734,15 +734,8 @@ public function updateReceptionStatus($params) {
 
       $detail                 = new PluginOrderOrder_Item();
       $plugin_order_orders_id = 0;
-      $ma                     = false;
 
-      // from MassiveAction process, we get ma object, so convert it into array
-      if (is_object($params)) {
-         $ma      = $params;
-         $params2 = (array) $params;
-      }
-
-      if (isset($params2['items'][__CLASS__])) {
+      if (isset($params->__get('items')[__CLASS__])) {
```

Comment:
Unless I am wrong, there is a method for this.
```suggestion
      if (isset($params->getItems()[__CLASS__])) {
```

Reply:
you're right -> fixed